Saucier720-app/src/app/recipes/FEC/recipe-card/recipe-card.component.ts

```diff
@@ -177,4 +188,59 @@ export class RecipeCardComponent implements OnInit {
     }
   }
 
+  holdTimer: any;
```

Comment:
We could define a Time type that is just an extension of number

Reply:
true